.github/workflows/honeypot-dapp.yml

```diff
@@ -0,0 +1,82 @@
+# yaml-language-server: =https://json.schemastore.org/github-workflow.json
+---
+name: "Build Honeypot DApp"
+on:
+    workflow_dispatch:
+    pull_request:
+        paths:
+            - "docker/**"
+            - "config/**"
+            - "docker-compose*.yml"
+            - "docker-bake.hcl"
+            - ".github/workflows/dapp-build.yml"
+            - ".github/workflows/honeypot-dapp.yml"
+            - "!**/*.md"
+            - "**/*.*"
+            - "tests/**"
+    push:
+        tags:
+            - v*
+        branches:
+            - develop
+            - adapt-robust-version
+permissions:
+    contents: read
+    packages: write
+    actions: write
+jobs:
+    build:
+        uses: ./.github/workflows/dapp-build.yml
+        with:
+            dapp-name: honeypot
+        secrets: inherit
+    test:
+        needs: build
+        runs-on: ubuntu-20.04
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v3
+              with:
+                  submodules: recursive
+                  token: ${{ secrets.CI_TOKEN }}
```

Comment:
Can remove the token.

If there is no submodule in the repo can also remove the `submodules: recursive`

Reply:
There is the "grpc interfaces" inside tests